pkg/webui/console/views/device-payload-formatters/index.js

```diff
@@ -35,6 +37,9 @@ import { selectSelectedDeviceId } from '../../store/selectors/device'
 
 import style from './device-payload-formatters.styl'
 
+const m = defineMessages({
+  infoText: 'Device payload formatter overrides application payload formatters, if set.',
```

Comment:
These payload formatters are executed on uplink messages from this device. If payload formatters are also set for the application, those are not executed for uplinks from this device.